modules/local/starfusion/build/main.nf

```diff
@@ -2,7 +2,9 @@ process STARFUSION_BUILD {
     tag 'star-fusion'
 
     conda "${moduleDir}/environment.yml"
-    container 'community.wave.seqera.io/library/dfam_hmmer_samtools_star-fusion_pruned:5694d82381bf039e'
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/f8/f804c086cee338c315fbc51faf0c37de360b047b47c33388716962eb3467b087/data':
+        'community.wave.seqera.io/library/dfam_hmmer_minimap2_samtools_pruned:bd39df228dad7086'}"
```

Comment:
Is there a reason why this PR (https://github.com/nf-core/rnafusion/pull/586) is not reviewed/merged first? I developed the working recipe for this one and its nf-test. It can allow us to track any subsequent changes to the modules.

Reply:
and this one: https://github.com/nf-core/rnafusion/pull/583

Reply:
Because I need singularity to test it on our system 😄 . I am using this PR to run the checks.

Reply:
Let's merge this one, it does update starfusion to 1.14 but your PRs main points are about the tests so I hope it is ok (no point is trying to fix dependencies for something we are going to update anyways)